.travis.yml

```diff
@@ -1,3 +1,10 @@
+dist: trusty
+sudo: required
+services:
+  - docker
+git:
+  depth: 2
```

Comment:
> Please note that if you use a depth of 1 and have a queue of jobs, Travis CI won’t build commits that are in the queue when you push a new commit.

This means that one can test only the last 2 commits to a branch and not more.
It is also related to: https://docs.travis-ci.com/user/customizing-the-build#Building-only-the-latest-commit

So of we have a branch where a lot of commits happen, only the last 2 are picked up.